From 1c7c0f71c9d22efda4156881eb187b8c69d1cca7 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 6 Feb 2017 14:28:19 -0500
Subject: [PATCH] Remove some extra "const" that gcc complains about.

One of these days I'll get these right.

Signed-off-by: Peter Jones <pjones@redhat.com>
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream commit 1c7c0f71c9d22e.

 src/include/efivar/efiboot-loadopt.h | 4 ++--
 src/loadopt.c                        | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/include/efivar/efiboot-loadopt.h b/src/include/efivar/efiboot-loadopt.h
index 07db5c4c53e3..efc29c69d47e 100644
--- a/src/include/efivar/efiboot-loadopt.h
+++ b/src/include/efivar/efiboot-loadopt.h
@@ -32,8 +32,8 @@ extern ssize_t efi_loadopt_create(uint8_t *buf, ssize_t size,
 
 extern efidp efi_loadopt_path(efi_load_option *opt, ssize_t limit)
 	__attribute__((__nonnull__ (1)));
-extern const unsigned char const * efi_loadopt_desc(efi_load_option *opt,
-						    ssize_t limit)
+extern const unsigned char * efi_loadopt_desc(efi_load_option *opt,
+					      ssize_t limit)
 	__attribute__((__visibility__ ("default")))
 	__attribute__((__nonnull__ (1)));
 extern uint32_t efi_loadopt_attrs(efi_load_option *opt)
diff --git a/src/loadopt.c b/src/loadopt.c
index a63ca792d2dc..ce889867fd29 100644
--- a/src/loadopt.c
+++ b/src/loadopt.c
@@ -357,7 +357,7 @@ teardown(void)
 
 __attribute__((__nonnull__ (1)))
 __attribute__((__visibility__ ("default")))
-const unsigned char const *
+const unsigned char *
 efi_loadopt_desc(efi_load_option *opt, ssize_t limit)
 {
 	if (last_desc) {
-- 
2.13.2

